src/clj/game/core/actions.clj

```diff
@@ -192,9 +192,9 @@
         abilities (:abilities cdef)
         ab (if (= ability (count abilities))
              ;; recurring credit abilities are not in the :abilities map and are implicit
-             {:msg "take 1 [Recurring Credits]" :req (req (> (:rec-counter card) 0))
+             {:msg "take 1 [Recurring Credits]" :req (req (> (get card :rec-counter 0) 0))
```

Comment:
You could also do `(:rec-counter card 0)`, as invoking a `:keyword` has an optional parameter which is the default: https://clojure.org/reference/data_structures#Keywords